src/components/Logotype.js

```diff
@@ -19,6 +19,7 @@ export const Logotype = (props) => {
 			<Image
 				style={{ maxWidth: width }}
 				src={useColorModeValue(inverted, primary)}
+				alt="Vader Logo"
```

Comment:
This can be changed to Vader Protocol.
In the future, I plan to distinguish between logo token use and protocol logo use.

Reply:
Should I make a new change for this now or keep it in mind for the future? :)